src/main.rs

```diff
@@ -0,0 +1,31 @@
+extern crate chrono;
+extern crate postgres;
+
+use std::{thread, env};
+use std::time::Duration;
+use postgres::{Connection, TlsMode};
+
+
+use chrono::Local;
+
+
+fn main() {
+    println!("Start daemon {}", Local::now());
+    // for example,  "postgresql://anton:12345@localhost:5432/l09";
+    let postgres_url = env::var("DBMETER_DB_URL").unwrap();
```

Comment:
Could you please rename env variable `DBMETER_DB_URL` to `DBMETER_DB_DSN`, because it is standard naming for that case. More details in https://en.wikipedia.org/wiki/Data_source_name